docs/release/release_0_6_3.md

```diff
@@ -25,8 +25,7 @@ A critical Windows-specific bug that caused Access Violation errors has been res
 
 ### Improved PySide6 Support 🛠️
 
-Napari now has improved support for PySide6 [(#7887)](https://github.com/napari/napari/pull/7887), resolving various compatibility issues and segfaults. This update enhances stability across platforms and prepares napari for the future as Qt5 approaches end-of-life.
-
+Napari now has improved support for PySide6 [(#7887)](https://github.com/napari/napari/pull/7887). We encourage plugin developers to test against PySide6, as that we will drop PySide2 as support ends with Python 3.10. If you are a plugin developer, or otherwise depend on napari and PySide2, please reach out on Zulip or Github.
```

Comment:
```suggestion
Napari now has improved support for PySide6 [(#7887)](https://github.com/napari/napari/pull/7887). We encourage plugin developers to test against PySide6, as we will soon drop PySide2, because its support ends with Python 3.10. If you are a plugin developer, or otherwise depend on napari and PySide2, please reach out on Zulip or Github.
```

Reply:
Maybe slightly clearer this way? What I am trying to convey is that folks should test with pyside6, because it's going to be fully supported going forward. Folks that depend on pyside2 should reach out.